ops/pebble.py

```diff
@@ -1563,13 +1561,15 @@ def _request(self,
         return raw_resp
 
     @staticmethod
-    def _ensure_content_type(headers: 'Message',
+    def _ensure_content_type(headers: email.message.Message,
                              expected: 'Literal["multipart/form-data", "application/json"]'):
         """Parse Content-Type header from headers and ensure it's equal to expected.
 
         Return a dict of any options in the header, e.g., {'boundary': ...}.
         """
-        ctype, options = cgi.parse_header(headers.get('Content-Type', ''))
+        ctype = headers.get_content_type()
+        params = headers.get_params() or {}
+        options = dict((key, value) for key, value in params if value)
```

Comment:
Maybe just my stylistic preference more than anything, but is there a reason why you chose to use the `dict()` built-in function instead of the squiggly brackets to generate the options dictionary?

```python
options = {key: value for key, value in params if value}
```

I find the squiggly brackets to be slightly more readable than `dict(...)`; I usually only use `dict(...)` if I am initializing an empty dictionary or converting another type to a dictionary.

Reply:
Good call, I prefer those too! Not sure why I used the old-school `dict()` method. Updated.